config/config.go

```diff
@@ -204,7 +204,7 @@ type Config struct {
 	RuleFiles      []string        `yaml:"rule_files,omitempty"`
 	ScrapeConfigs  []*ScrapeConfig `yaml:"scrape_configs,omitempty"`
 
-	RemoteWriteConfig RemoteWriteConfig `yaml:"remote_write,omitempty"`
```

Comment:
I was going to say `remote_storage` too, but was anticipating that @brian-brazil would say that it's not necessarily storage which sits behind the interface :)

I think I'll keep it as it is, but remove the `_config` suffix?